Kickstarter-iOS/Locales/Base.lproj/Localizable.strings

```diff
@@ -121,6 +121,7 @@
 "Continue_with_quantity_count_add_ons.two" = "Continue with %{quantity_count} add-ons";
 "Continue_with_quantity_count_add_ons.zero" = "Continue with %{quantity_count} add-ons";
 "Continue_with_this_reward" = "Continue with this reward?";
+"Couldn't_post" = "Couldn't post. Tap to retry";
```

Comment:
Is it OK for this apostrophe to be here? The function generated in the `Strings` file required a manual removal of it before compiling.

Reply:
Nvm, going to remove this apostrophe and update.

Reply:
[This PR](https://github.com/kickstarter/kickstarter/pull/21788) should handle the issue of `make strings` returning the string with the apostrophe again.